### DISCUSSION TITLE
How to make responsive desktop application using uno platform?

### DISCUSSION THREAD

**Discussion by cschandancs** [unoplatform/uno#8201] [Q&A]
I want to make responsive desktop application which can run on different platform mainly(Linux, Windows) as well as different screen size.
for eg. there is a grid in which 2x2 column and row. 
I want to put control like Textbox ,Label inside  grid (column, row) which can be responsive based on screen size.
And also application only for windows and Linux not for android , ios. I want that dependency (android, ios) should remove while creating project.

**Comment by jeromelaban** [comment]
To create a responsive app, the base is generally to use [AdaptiveTrigger](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.AdaptiveTrigger) (some [more documentation here](https://docs.microsoft.com/en-us/windows/apps/design/layout/layouts-with-xaml)). You can update your UI layout based on that.

For the other platforms, you can remove the unused projects and code after you've created your application. You only need to keep the "uwp" and "shared" projects, if you created your project from Visual Studio.

If you created your project from `dotnet new`, please see the parameters [here](https://platform.uno/docs/articles/get-started-dotnet-new.html#uno-platform-blank-application).
